examples/hra_dreambooth/train_dreambooth.sh

```diff
@@ -0,0 +1,191 @@
+    "a red ${unique_token} ${class_token}"
+    "a purple ${unique_token} ${class_token}"
+    "a shiny ${unique_token} ${class_token}"
+    "a wet ${unique_token} ${class_token}"
+    "a cube shaped ${unique_token} ${class_token}"
+  )
+
+  prompt_test_list=(
+    "a ${class_token} in the jungle"
+    "a ${class_token} in the snow"
+    "a ${class_token} on the beach"
+    "a ${class_token} on a cobblestone street"
+    "a ${class_token} on top of pink fabric"
+    "a ${class_token} on top of a wooden floor"
+    "a ${class_token} with a city in the background"
+    "a ${class_token} with a mountain in the background"
+    "a ${class_token} with a blue house in the background"
+    "a ${class_token} on top of a purple rug in a forest"
+    "a ${class_token} wearing a red hat"
+    "a ${class_token} wearing a santa hat"
+    "a ${class_token} wearing a rainbow scarf"
+    "a ${class_token} wearing a black top hat and a monocle"
+    "a ${class_token} in a chef outfit"
+    "a ${class_token} in a firefighter outfit"
+    "a ${class_token} in a police outfit"
+    "a ${class_token} wearing pink glasses"
+    "a ${class_token} wearing a yellow shirt"
+    "a ${class_token} in a purple wizard outfit"
+    "a red ${class_token}"
+    "a purple ${class_token}"
+    "a shiny ${class_token}"
+    "a wet ${class_token}"
+    "a cube shaped ${class_token}"
+  )
+fi
+
+validation_prompt=${prompt_list[$prompt_idx]}
+test_prompt=${prompt_test_list[$prompt_idx]}
+name="${selected_subject}-${prompt_idx}"
+instance_prompt="a photo of ${unique_token} ${class_token}"
+class_prompt="a photo of ${class_token}"
+
+export OUTPUT_DIR="logs/${name}"
+export INSTANCE_DIR="dreambooth/dataset/${selected_subject}"
+export CLASS_DIR="class_data/${class_token}"
+
+if [ -d "$OUTPUT_DIR" ]; then
+  echo "This directory already exists. Please delete it and try again. : $OUTPUT_DIR"
+else
+  echo "class_token: $class_token, prompt: $validation_prompt"
+  accelerate launch train_dreambooth.py \
+    --pretrained_model_name_or_path=$MODEL_NAME  \
+    --instance_data_dir=$INSTANCE_DIR \
+    --class_data_dir="$CLASS_DIR" \
+    --output_dir="$OUTPUT_DIR" \
+    --instance_prompt="$instance_prompt" \
+    --with_prior_preservation \
+    --prior_loss_weight=1.0 \
+    --class_prompt="$class_prompt" \
+    --resolution=512 \
+    --train_batch_size=1 \
+    --gradient_accumulation_steps=1 \
+    --learning_rate=1e-3 \
+    --report_to="wandb" \
+    --lr_scheduler="constant" \
+    --lr_warmup_steps=0 \
+    --max_train_steps=1705 \
+    --validation_prompt="$validation_prompt" \
+    --seed="0" \
+    --num_class_images=200 \
+    --checkpointing_steps=200 \
+    --validation_steps=200 \
+    --train_text_encoder \
+    --use_hra \
+    --hra_r=$hra_r 
+
+  echo "Please fill in the following parameters in the hra_dreambooth_inference.ipynb"
+  echo "BASE_MODEL_NAME = $MODEL_NAME"
+  echo "ADAPTER_MODEL_PATH = $OUTPUT_DIR"
+  echo "PROMPT = $instance_prompt"
```

Comment:
Just a suggestion, this way users can copy&paste into the notebook without having to edit the code.

```suggestion
  echo "BASE_MODEL_NAME = '$MODEL_NAME'"
  echo "ADAPTER_MODEL_PATH = '$OUTPUT_DIR'"
  echo "PROMPT = '$instance_prompt'"
```

Reply:
Done.